src/features/broker-protection/actions/extract.js

```diff
@@ -41,13 +41,21 @@ export function extract (action, userData) {
         .filter(x => x.result === true)
         .map(x => aggregateFields(x.scrapedData))
 
-    return new SuccessResponse({ actionID: action.id, actionType: action.actionType, response: filtered })
+    return new SuccessResponse({
+        actionID: action.id,
+        actionType: action.actionType,
+        response: filtered,
+        meta: {
+            userData,
+            extractResults: extractResult.results
+        }
+    })
```

Comment:
We don't alter the core 'response' here (which would cause native to have to change), instead we provide an additional field